WordPress/src/main/java/org/wordpress/android/util/image/ImageManager.kt

```diff
@@ -90,51 +98,7 @@ class ImageManager @Inject constructor(val placeholderManager: ImagePlaceholderM
 
     @Deprecated("Object for backward compatibility with code which doesn't support DI")
     companion object {
-        @JvmStatic
-        @Deprecated("Use injected ImageManager",
-                ReplaceWith("imageManager.load(imageView, imgUrl, placeholder, scaleType)",
-                        "org.wordpress.android.util.image.ImageManager"))
-        fun loadImage(
-            imageView: ImageView,
-            imageType: ImageType,
-            imgUrl: String,
-            scaleType: ImageView.ScaleType
-        ) {
-            ImageManager(ImagePlaceholderManager()).load(imageView, imageType, imgUrl, scaleType)
-        }
-
-        @JvmStatic
-        @Deprecated("Use injected ImageManager",
-                ReplaceWith("imageManager.load(imageView, bitmap, scaleType)",
-                        "org.wordpress.android.util.image.ImageManager"))
-        @JvmOverloads
-        fun loadImage(imageView: ImageView, bitmap: Bitmap, scaleType: ImageView.ScaleType = CENTER) {
-            ImageManager(ImagePlaceholderManager()).load(imageView, bitmap, scaleType)
-        }
-
-        @JvmStatic
-        @Deprecated("Use injected ImageManager",
-                ReplaceWith("imageManager.load(imageView, drawable, scaleType)",
-                        "org.wordpress.android.util.image.ImageManager"))
-        @JvmOverloads
-        fun loadImage(imageView: ImageView, drawable: Drawable, scaleType: ImageView.ScaleType = CENTER) {
-            ImageManager(ImagePlaceholderManager()).load(imageView, drawable, scaleType)
-        }
-
-        @JvmStatic
-        @Deprecated("Use injected ImageManager",
-                ReplaceWith("imageManager.loadIntoCircle(imageView, imgType, imgUrl)",
-                        "org.wordpress.android.util.image.ImageManager"))
-        fun loadImageIntoCircle(imageView: ImageView, imageType: ImageType, imgUrl: String) {
-            ImageManager(ImagePlaceholderManager()).loadIntoCircle(imageView, imageType, imgUrl)
-        }
-
-        @JvmStatic
-        @Deprecated("Use injected ImageManager",
-                ReplaceWith("imageManager.clear(imageView)",
-                        "org.wordpress.android.util.image.ImageManager"))
-        fun clear(imageView: ImageView) {
-            ImageManager(ImagePlaceholderManager()).cancelRequestAndClearImageView(imageView)
-        }
+        @Deprecated("Use injected ImageManager")
```

Comment:
You can add `@JvmStatic` to avoid having to call it by `ImageManager.Companion.getInstance()` and use `ImageManager.getInstance()` instead.

Reply:
I've completely forgotten about that annotation. Thanks!